src/tls13.c

```diff
@@ -5797,6 +5799,8 @@ static int FindPsk(WOLFSSL* ssl, PreSharedKey* psk, const byte* suite, int* err)
     }
 
     *err = ret;
+    WOLFSSL_LEAVE("FindPsk", found);
```

Comment:
Is this a trick to also log "found"?

Reply:
Yes I needed to figure out what the FindPSK outcome was. If you don't like it, I can remove it.